### PR TITLE
Add NAP-5: proposal for an updated napari logo

### PR DESCRIPTION
**Pull request by jni**
# Description

Hi folks! Following a private discussion with the core team, I would like to propose a new logo for napari. The logo tries to keep the shape of the original island, but making it more stylized. It uses the Golden ratio ϕ for proportions (which results in a lot of cool math 😃), and is **generated entirely within napari.**

The NAP has all of the details. One important aspect is that the proposal is principally about the *shape* of the logo. I'm completely open to playing around with:

- the final colours
- whether or not to have a "sun reflection" on the ocean
- whether or not to add a drop shadow

but I'd love it if someone else wanted to play with those. 😅

Please read the NAP for details but here is a comparison screenshot:

napari-logo-old-new

imho the new version looks more modern and friendly, almost jolly. 😃 But, I look forward to your comments! 🫣😬

### CODE REVIEW

**Comment by jni** [comment]
Ah, since GH won't display LFS files, here is the logo with guides, showing the proportions of ϕ:

napari-logo-3-with-guides

**Comment by Czaki** [comment]
it is possible to prepare and post "flat" version of the current logo? To see how many changes come from removing the gradients, and how many from changing proportions?

**Comment by junxini** [comment]
Wow Juan, a lot of consideration and careful planning went into this. It's mind-blowing and I love it!

> whether or not to have a "sun reflection" on the ocean

looking at the apps I use daily for inspiration, I'd vote not having a sun reflection on the ocean

**Comment by jni** [comment]
> it is possible to prepare and post "flat" version of the current logo?

LOL @Czaki, I was sitting around this weekend thinking, man I can't be bothered fiddling with that... Then I happened to glance at my mugs shelf, and to my astonishment it was right there!!! 😂 

4500F35E-334A-4033-9632-EBCF579BE9AE_1_105_c

Courtesy of CZI and the napari-hub team! 😂 Not only is it flat but I feel like the colours are pretty darn close to the ones in mine! 😂

File: docs/naps/5-new-logo.md
Comment (kevinyamauchi) [inline docs/naps/5-new-logo.md:161]: I agree that tiling is a nice-to-have, but as you said, the scientific python ecosystem stickers have not really gone with a tile-able standard. Maybe a hexagonal packing sticker standard would be a good [SPEC](https://scientific-python.org/news/specs-announced-scipy2021/) for the scientific python community ;)

File: docs/naps/5-new-logo.md
Comment (kevinyamauchi) [inline docs/naps/5-new-logo.md:127]: I agree it could make sense to separate colors from shapes. However,  I think it would be useful to list the RGB colors used for the different parts of the logo for people developing other materials (e.g., posters/slides).

File: docs/naps/5-new-logo.md
Comment (kevinyamauchi) [inline docs/naps/5-new-logo.md:113]: 😍

**Review by kevinyamauchi — APPROVED** [review]
This looks good to me! I am +1 for adding this in draft status (minor comments below).

Before merging, I think somebody else with a better understanding git lfs should make sure the screenshots were added correctly.

**Comment by Czaki** [comment]
For me, the flat version of the current logo looks better. But is personal and subjective.

File: docs/naps/5-new-logo.md
Comment (jni) [reply docs/naps/5-new-logo.md:127]: I mean, they'll have the code... 😉

Comment (GenevieveBuckley) [reply docs/naps/5-new-logo.md:127]: People should absolutely just be able to download the logos from a page on napari.org, not be forced to generate them themselves @jni

Here's one example (I think this could be better if it also included a theme for powerpoint and google slides): https://www.dask.org/brand-guide

**Comment by GenevieveBuckley** [comment]
I like the slightly irregular shape, it feels more organic (like the biology most of us work on). TBH, I'd always assumed it was supposed to be an artist's rendition of a cell undergoing mitosis.

Honestly I feel like there's no particular reason to value being able to generate napari's logo from within napari. I think you just really like the *idea* of that (which is fine!) but there's no advantage in it.

Mostly I don't want to spend a lot of time discussing this, so I'm going to mute this thread and won't be available for follow up back & forth.

**Comment by liu-ziyang** [comment]
No one asked me to be here, but here are my subjective 2 cents:
I also like the old shape more, I can echo all the reasons given by @GenevieveBuckley. That being said, I am not sure to what degree this anchoring bias comes from the familiarity with the old logo. 

I do like the coloring in the new logo much more, it is brighter and more cheering to me, and the contrast between the background is more comfortable to my eye. The gradient of shade for the lake in the old logo looks dizzy to me for some reason.

Maybe alternatively, we can try to find math that generates the old logo or a close match that resembles it?

**Comment by andy-sweet** [comment]
There haven't been many comments in a while, so I'll probably merge this after 24 hours unless I hear otherwise.

**Comment by jni** [comment]
Thanks @andy-sweet ! There's been a few comments that I'll need to note in the discussion section of the NAP, but I can do that in later PRs. I'll try to make a few more tweaks based on feedback.

**Pull request merged**
